README.md

```diff
@@ -39,45 +41,51 @@ fn health() -> HttpResponse {
 }
 
 fn main() -> std::io::Result<()> {
-    let prometheus = PrometheusMetrics::new("api", "/metrics");
+    let mut labels = HashMap::new();
+    labels.insert("label1".to_string(), "value1".to_string());
+    let prometheus = PrometheusMetrics::new("api", Some("/metrics"), Some(labels));
     HttpServer::new(move || {
         App::new()
             .wrap(prometheus.clone())
             .service(web::resource("/health").to(health))
     })
     .bind("127.0.0.1:8080")?
     .run();
+    }
```

Comment:
Extraneous `}` I believe.

Reply:
You're right. Fixed.

Reply:
Fwiw, it would be nice to extract (rather than copy) the examples in the README.md from the documentation that's in `src/lib.rs` because this code gets tested and is thus guaranteed to work.